Pipfile

```diff
@@ -35,3 +35,7 @@ python_version = "3.8"
 
 [pipenv]
 allow_prereleases = true
+
+[scripts]
```

Comment:
Cool! I didn't know about these.

Reply:
Yeah, I hunted it down because I liked a similar feature in yarn, and had just run a stupidly long command (in a toy project, nothing work related).

Probably  find that bundler does something similar, just so used to bundle exec I don't think to check. Although in bundlers case they may have decided that binstubs are sufficient.